app/controllers/personal_access_tokens_controller.rb

```diff
@@ -0,0 +1,38 @@
+class PersonalAccessTokensController < ApplicationController
+  include Foreman::Controller::Parameters::PersonalAccessToken
+  include Foreman::Controller::SshKeysCommon
+  include Foreman::Controller::ActionPermissionDsl
+
+  before_action :find_resource, :only => [:revoke]
+
+  define_action_permission 'revoke', :revoke
+
+  def new
+    @personal_access_token = PersonalAccessToken.new
+  end
+
+  def create
+    @personal_access_token = PersonalAccessToken.new(personal_access_token_params.merge(:user => @user))
+    token_value = @personal_access_token.generate_token
+    if @personal_access_token.save
+      flash[:personal_access_token] = token_value
```

Comment:
@ohadlevy: I need to pass the data from the `personal_access_token` controller (which generates the token) to `users_controller` (which displays the token + index). The `personal_access_token` controller could just render a page that displays the token and the flash would not be necessary. However, I wanted to avoid this kind of page. Another option would be to do all this with client side javascript. But I'm not really a frontend guy, you know? :-)